### ISSUE TITLE
Anchor dialog should allow classes to be defined

### ISSUE THREAD

**Issue by mausolos** [ckeditor/ckeditor-dev#485]
## Are you reporting a feature or a bug?
Feature request

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)

Couldn't find a duplicate or related issue (searched for "anchor" and "scroll")

## Provide detailed reproduction steps (if any)

1. Create a new anchor in some content
2. View the resulting content

### Expected result

Should scroll to the correct location. If current behavior is "as designed", then a class should be added to the anchor that CKEditor generates so that developers and themers can target it for something like this:

`.ckeditor-anchor {
  display: block;
  position: relative;
  top: -2.2rem;
}
`

### Actual result

Some front-end themes that render the resulting content will scroll farther down the page, so text intended to be displayed (header elements, for instance) do not show at the top of the user's page

## Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by jswiderski** [comment]
Good idea. At the moment anchor plugins inserts `cke_anchor` but this class is not populated to resulting HTML.

It would be nice to have configuration setting for that so that user could specify his own unique class name. I would also go a step further and ask for configuration setting for anchor image so that users could customize the editor's anchor.
